runtime/frequency/src/lib.rs

```diff
@@ -130,16 +130,14 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 		#[cfg(not(feature = "frequency"))]
 		{
 			match call {
-				// Utility Calls are blocked. Issue #599
-				RuntimeCall::Utility(..) => false,
+				RuntimeCall::Utility(..) => Self::is_utility_call_allowed(call),
 				_ => true,
 			}
 		}
 		#[cfg(feature = "frequency")]
 		{
 			match call {
-				// Utility Calls are blocked. Issue #599
-				RuntimeCall::Utility(..) => false,
+				RuntimeCall::Utility(..) => Self::is_utility_call_allowed(call),
```

Comment:
Following implementation is only invoked for utility calls, so it wont affect any other pallet calls